templates/index.html

```diff
@@ -272,25 +272,7 @@ <h4 class="title"></h4>
             <p class="mainStreet" style="margin:0"></p>
             <p class="secondaryStreet" style="margin:0"></p>
             <p class="junction" style="margin:0"></p>
-            <p class="involved_type" style="margin:0"></p>
```

Comment:
Since @OmerSchechter is going to take this and use it in the new accordion design for the marker I don't think it is required..